src/plugins/wizard/public/visualizations/metric/components/metric_viz_options.tsx

```diff
@@ -5,8 +5,7 @@
 
 import React, { useCallback } from 'react';
 import { i18n } from '@osd/i18n';
-import { EuiButtonGroup, EuiFormRow, EuiPanel, EuiSpacer, EuiTitle } from '@elastic/eui';
-import { FormattedMessage } from 'react-intl';
+import { EuiButtonGroup, EuiFormRow } from '@elastic/eui';
```

Comment:
can we explicitly use oui now? both should work right?

Reply:
I tried that but Typescript was complaining. So i moved the issue out for later